src/features/messaging-next/ConversationPanel.module.css

```
@@ -81,6 +81,44 @@
   text-wrap: pretty;
 }
 
+.msg:has(.filePreviewImg) {
+  background: transparent;
+
+  padding: 0;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)